src/Bicep.Core/Registry/OciModuleRegistry.cs

```diff
@@ -240,13 +242,16 @@ public override async Task PublishModule(OciArtifactModuleReference moduleRefere
             }
         }
 
-        protected override void WriteModuleContent(OciArtifactModuleReference reference, OciArtifactResult result)
+        // Writes the contents of the downloaded module into the local cache
+        protected override void DownloadToCache(OciArtifactModuleReference reference, OciArtifactResult result)
```

Comment:
>DownloadToCache [](http://example.com/codeflow?start=32&length=15)

The new method name is misleading. It doesn't do any downloading. Probably better to rename to `WriteModuleContentToCache()` instead.

Reply:
done